General/general_shacl.ttl

```diff
@@ -0,0 +1,133 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+general:GeneralShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node gx:DataResource ;
+            sh:order 1 ;
+            sh:path general:DataResource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:LinksShape ;
+            sh:order 4 ;
+            sh:path general:Links ] ;
+    sh:targetClass general:General .
+
+general:LinksShape a sh:NodeShape ;
+    sh:property [ skos:example "link to screenshot, video, routing" ;
+            sh:description "Reference to screenshot data"@en ;
+            sh:message "Validation of media failed!"@en ;
+            sh:name "media"@en ;
+            sh:node general:LinkShape ;
+            sh:order 2 ;
+            sh:path general:media ],
+        [ skos:example "???" ;
+            sh:datatype xsd:string ;
+            sh:description "contract id -> is generated/assigned later via the EDC"@en ;
```

Comment:
I recommend not using implementations here like EDC

Reply:
could be something like:
"contract information in regards to the data exchange component."

Reply:
as discussed I have removed the contract id - wanted to regulate the EDC

Reply:
Contract ID should be there and not being deleted. Once we have the contract ID in the gx vocabulay we can delete it from here.

@jdsika 's problem was the term "EDC" in the description since it is only one possibility of executing data transfer and there may be different implementations.

One possible description could be
"contract information in regards to the data exchange component."
as stated above.

Reply:
I have added it back to the Data category (together with size) and adjusted the description. The example and the data type are open!

Reply:
@MircoNierenz I would set the datatype to string as it is already.
Example (from a real use case): contract_zcdkr7kqd47y0w5b4tg91w1etw
I would prefer `contractId` instead of `contractID`

Reply:
change name. done!